css/style.css

```diff
@@ -1,4 +1,38 @@
-@import url(https://fonts.googleapis.com/css?family=Open+Sans:300,400,400italic,700);
+/*
+ * Muli is licensed under SIL OFL Version 1.1
+ * See /assets/logos/font/Muli/OFL.txt file for full information.
+ */
+@font-face {
+    font-family: 'Muli';
+    src: url('/assets/logos/font/Muli/Muli-Bold.woff') format('woff');
+    font-weight: 700;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Muli';
+    src: url('/assets/logos/font/Muli/Muli-Regular.woff') format('woff');
+    font-weight: 400;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Muli';
+    src: url('/assets/logos/font/Muli/Muli-Light.woff') format('woff');
```

Comment:
Looks like we only use 300 for `h2`s, I'm happy to remove that value and get rid of that extra file.